modules/network-observability-filtering-ebpf-rule.adoc

```diff
@@ -57,18 +70,23 @@ spec:
   deploymentModel: Direct
   agent:
     type: eBPF
-    ebpf:        
+    ebpf:
+      privileged: true <1>
+      features:
+        - PacketDrop <2>
       flowFilter:
-        action: Accept  <1>
-        cidr: 0.0.0.0/0 <2>
-        protocol: TCP
-        direction: Egress
-        sourcePort: 100
-        peerIP: 192.168.127.12 <3>
-        enable: true    <4>
-----       
-<1> You can `Accept` flows based on the criteria in the `flowFilter` specification.
-<2> The `cidr` value of `0.0.0.0/0` matches against any IP address.
-<3> See flows after `peerIP` is configured with `192.168.127.12`. 
-<4> You must set `spec.agent.ebpf.flowFilter.enable` to `true` to enable the feature.
-====
\ No newline at end of file
+        enable: true <3>
+        rules:
+        - action: Accept <4>
+          cidr: 172.30.0.0/16
+          pktDrops: true <5>
+----
+<1> To enable packet drops, you must set `spec.agent.ebpf.privileged` to `true`.
+<2> You can start reporting the packet drops of each network flow by listing the `PacketDrop` parameter in the `spec.agent.ebpf.features` specification list.
+<3> To enable eBPF flow filtering, you must set `spec.agent.ebpf.flowFilter.enable` to `true`.
+<4> The required `action` parameter specifies the action that is taken for the flow filter rule. Possible values are `Accept` or `Reject`.
+<5> Set `pktDrops` to `true` to filter flows containing drops.
+
+By default, all other flows are rejected.
```

Comment:
Same comments on this one for the most part. List items could be more parallel, "By default, all other flows are rejected." on a new line stands out. 

In this case, the "By default, all other flows are rejected." bit is also formatted differently than the one above - not sure which is more correct. Does that statement mean to apply to the last callout, or to the YAML config as a whole?